packages/@okta/vuepress-site/code/library-versions/index.md

```diff
@@ -58,6 +58,8 @@ This table lists the current (and historical) major versions of our open-source
 | [Android AppAuth SDK][appauth-android]                          | 0.2.4                              | Retiring on 2020-03-20                                 |
 | [Android OIDC][oidc-android]                          | 1.0.1                              | Stable
```

Comment:
As long as we're updating this page, can we make this "Stable" bold like the rest?